spec/linkedList.js

```diff
@@ -0,0 +1,175 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import LinkedList from '../src/linkedList'
+
+chai.use(chaiChange)
+
+before( () => {
+  let linkedList = new LinkedList()
+  linkedList.insert('apple')
+  linkedList.insert('banana')
+})
+describe('LinkedList', () => {
+
+
+  it('exists', () => {
+
+    expect(LinkedList).to.be.a('function')
+  })
+
+    it('returns head node', () => {
+      const linkedList = new LinkedList()
+      linkedList.insert('apple')
+      linkedList.insert('banana')
+
+      expect(linkedList.getHeadNode().data).to.equal('apple')
+    })
+
+    it('returns tail node', () => {
+      const linkedList = new LinkedList()
+      linkedList.insert('apple')
+      linkedList.insert('banana')
+
+      expect(linkedList.getTailNode().data).to.equal('banana')
+
```

Comment:
Everywhere, I'm not going to comment them all.  It almost doesn't matter which way you pick, just be maximum consistent before submitting PR